packages/back-end/src/controllers/experiments.ts

```diff
@@ -1624,7 +1625,8 @@ export async function postSnapshot(
     regressionAdjusted: !!regressionAdjustmentEnabled,
     dimensions: dimension ? [dimension] : [],
     sequentialTesting: !!sequentialTestingEnabled,
-    sequentialTestingTuningParameter,
+    sequentialTestingTuningParameter: sequentialTestingTuningParameter,
+    baselineVariation: experiment.variations[0].name,
```

Comment:
how is baseline switching implemented? is it through experiment.variations[] reordering (as it seems that 0 is hardcoded)?

Reply:
also, is "name" a safe, unique, immutable identifier?